cirq/google/op_serializer.py

```diff
@@ -31,17 +33,11 @@
 Gate = TypeVar('Gate', bound=ops.Gate)
 
 
-class SerializingArg(
-        NamedTuple(
-            'SerializingArg',
-            [('serialized_name', str),
-             ('serialized_type', Type[arg_func_langs.ArgValue]),
-             ('gate_getter',
-              Union[str, Callable[['cirq.Gate'], arg_func_langs.ArgValue]]),
-             ('required', bool)])):
+@dataclass
```

Comment:
consider `@dataclass(frozen=True)` for immutability and hashability

Reply:
Done.